sqlglot/expressions.py

```diff
@@ -1697,6 +1697,11 @@ class Length(Func):
     pass
 
 
+class Levenshtein(Func):
+    arg_types = {"this": True, "expression": False}
+    is_var_len_args = True
```

Comment:
why do you have var len args set? is it not always a and b?

Reply:
rm'd, thanks